homeassistant/components/vicare/climate.py

```diff
@@ -107,18 +117,15 @@ async def async_setup_entry(
     """Set up the ViCare climate platform."""
     entities = []
     api = hass.data[DOMAIN][config_entry.entry_id][VICARE_API]
+    device_config = hass.data[DOMAIN][config_entry.entry_id][VICARE_DEVICE_CONFIG]
     circuits = await hass.async_add_executor_job(_get_circuits, api)
 
     for circuit in circuits:
-        suffix = ""
-        if len(circuits) > 1:
-            suffix = f" {circuit.id}"
-
         entity = ViCareClimate(
-            f"Heating{suffix}",
             api,
             circuit,
-            hass.data[DOMAIN][config_entry.entry_id][VICARE_DEVICE_CONFIG],
+            device_config,
+            ViCareClimateEntityDescription(key="heating", translation_key="heating"),
```

Comment:
Why not pass in the translation key? As in, it seems a bit overkill to use entity descriptions here, when they are basically not used?

Reply:
On the other platforms the `key` is also used, but this is not the case here. So I will replace this with a simple string.